js/Utils.js

```diff
@@ -32,6 +32,20 @@ export default {
     return cssStr;
   },
 
+  splitBackgroundColorFromOtherStyles(originalStyle: string): Object {
+    let style = originalStyle;
+    const backgroundColorIndex = style.indexOf('background-color');
+    let backgroundColorStyle = '';
+    if (backgroundColorIndex > -1) {
+      backgroundColorStyle = style.substr(backgroundColorIndex, style.indexOf(';', backgroundColorIndex) - (backgroundColorIndex - 1)); // add backgroundColorStyle to div container
+      style = style.replace(backgroundColorStyle, ''); // remove backgroundColorStyle from span
+    }
+    return {
+      backgroundColorStyle,
+      style,
+    };
+  },
+
```

Comment:
I would write the util for more general propose - something like:

Function:
```js
extractFromStyles(originalStyle: string, prop: string): Object {
    ...
    return {
      extractedStyle,
      style,
    };
  },
```
Usage:
```js
const splitedStyle = Utils.extractFromStyles(style, 'background-color');
```

Reply:
done